src/util/DeviceFingerprint.js

```diff
@@ -10,28 +10,36 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* eslint complexity: [2, 7] */
+/* eslint complexity: [2, 8] */
 
 define(['vendor/lib/q', 'okta/jquery'], function (Q, $) {
 
   return {
+    __getUserAgent: function() {
+      return navigator.userAgent;
+    },
+    _isMessageFromCorrectSource: function($iframe, event) {
```

Comment:
either add this one with `__` as a suffix or change `__getUserAgent` to have only one `_`. I vote for 2 underscores...

Although, now that I think about this, we don't really have a pattern here in the widget repo for naming "internal" or "private" functions. so I will probably just leave the names without any suffix. Plus they are not actually "private", since we are also returning them.